### DISCUSSION TITLE
Show video

### DISCUSSION THREAD

**Discussion by ghost** [emilk/egui#804] [Q&A]
Is it possible to somehow display video from storage or stream?

**Comment by emilk** [comment]
If you can find a library that can decode a video, then upload each new frame to a texture and show it using `egui::Image` (https://github.com/emilk/egui#how-do-i-render-3d-stuff-in-an-egui-area)

**Comment by coderedart** [comment]
considering audio and synchronization issues, an alternative is to use a native video player with https://github.com/Byron/open-rs . or bindings like https://github.com/Cobrand/mpv-rs or https://github.com/garkimasera/vlc-rs
